mne/decoding/transformer.py

```diff
@@ -172,6 +172,34 @@ def transform(self, epochs_data, y=None):
         return _sklearn_reshape_apply(self._scaler.transform, True,
                                       epochs_data, y)
 
+    def fit_transform(self, epochs_data, y=None):
+        """Fit to data, then transform it.
+
+        Fits transformer to epochs_data and y and returns a transformed version 
+        of epochs_data.
+
+        Parameters
+        ----------
+        epochs_data : array, shape (n_epochs, n_channels, n_times)
+            The data.
+        y : None | array, shape (n_epochs,)
+            The label for each epoch.
+            If None not used. Defaults to None.
```

Comment:
Hello @kingjr, sorry for late reply as I'm having exams in my college.
I kept this line by following the doc of `transform()` prior to this but I think it should be replaced by 
`If y not used defaults to None` or by `Defaults to None`.
What about your opinion ?